examples/playwright/package.json

```diff
@@ -13,11 +13,11 @@
   "homepage": "https://github.com/eclipse-theia/theia",
   "scripts": {
     "prepare": "yarn clean && yarn build",
-    "clean": "rimraf lib",
+    "clean": "rimraf lib *.tsbuildinfo",
     "build": "tsc --incremental && yarn lint && npx playwright install chromium",
```

Comment:
For another time: as part of https://github.com/eclipse-theia/theia/pull/9710, linting was removed from the `prepare` stage of our builds, but it still runs for Playwright. Since linting can be time consuming, it would be preferable to move it to its own command.

Reply:
Ah, thanks for the hint! I appended a change to remove linting from the `yarn build` script.
As I see `yarn lint` will be explicitly invoked during the CI build. So we can safely remove it from the `yarn build` script.
The `playwright` package has an explicit lint script already, so that should already be fine.